packages/amplication-git-utils/src/git/git.service.ts

```diff
@@ -1,207 +0,0 @@
-import { Injectable } from "@nestjs/common";
```

Comment:
This file should still be used and it should be the entry point for the lib. The clients who use the lib should not be aware of the internal provider/factory implementation.